app/views/admin/partners/_form.html.erb

```diff
@@ -124,6 +124,44 @@
         <% end %>
       </div>
     </div>
+    <div class="col-md-6">
+      <h3>Partnership Contact</h3>
+      <p><em>If different from public contact:</em> The contact person for PlaceCal.</p>
+      <%= f.input :partner_name, class: "form-control" %>
+      <%= f.input :partner_email, class: "form-control" %>
+      <%= f.input :partner_phone, class: "form-control" %>
+    </div>
+  </div>
+  <hr>
+  <div class="row">
+    <div class="col-lg-8 col-md-12">
+      <h2>Opening times</h2>
+      <%= render 'opening_times', f: f %>
+    </div>
+  </div>
+  <hr>
+  <h2>Tags</h2>
+
+  <p>What partnerships, categories and facilities does this partner have or belong to?</p>
+  <p>Partners may have up to 3 category tags.</p>
+
+  <%= f.association :tags,
+      label: false,
+      collection: options_for_partner_tags,
+      input_html: { class: 'form-check', data: { controller: "select2" } } %>
+
+  <hr>
+  <br>
+  <div class="row">
+    <div class="col-sm-12">
+      <%= f.submit "Save Partner", class: "btn btn-primary btn-lg" %><br><br><br>
+      <% if policy(@partner).destroy? && !@partner.new_record? %>
+        <%= link_to "Delete Partner", @partner, method: :delete, class: "btn btn-danger btn-sm", id: 'destroy-partner'  %>
+      <% else %>
+        <p>You do not have permission to delete this partner, please contact <%= mail_to "info@placecal.org", "info@placecal.org" %> if you need to delete this partner.</p>
```

Comment:
uh- this text shows up even if its a new record. you probably want something like
```
<% if !@partner.new_record? %>
  <% if policy(@partner).destroy? 
    <%= link_to "Delete Partner", @partner, method: :delete, class: "btn btn-danger btn-sm", id: 'destroy-partner'  %>
  <% else %>
    <p>You do not have permission to delete this partner, please contact <%= mail_to "info@placecal.org", "info@placecal.org" %> if you need to delete this partner.</p>
  <% end %>
<% end %>
```

Reply:
ohhhhhh it's the else clause. doh.